changelogs/fragments/78678-add-a-g-install-offline.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - ansible-galaxy collection install - add an --offline option to prevent querying distribution servers (https://github.com/ansible/ansible/issues/77443).
```

Comment:
```suggestion
  - ``ansible-galaxy collection install`` - add an ``--offline`` option to prevent querying distribution servers (https://github.com/ansible/ansible/issues/77443).
```

Reply:
Thanks :+1: